### PR TITLE
Cleanup some templating, remove resorting of show lists and iterable …

### LINKED ISSUE #3432: [APP SUBMITTED]: unsupported type for timedelta days component: tuple
### INFO
Python Version: **2.7.9 (default, Sep 17 2016, 20:26:04) [GCC 4.9.2]**
Operating System: **Linux-4.9.14-v7+-armv7l-with-debian-8.0**
Locale: UTF-8
Branch: **master**
Commit: SickRage/SickRage@78143873f776c0506c340c0eb6bab2f166952968
No Log available with ERRORS:
### ERROR
```
TORNADO :: [7814387] API :: unsupported type for timedelta days component: tuple
```
---
_STAFF NOTIFIED_: @SickRage/owners @SickRage/moderators

### PR DESCRIPTION
**Pull request by miigotu**
Don't display shows in the showList if they are in the queue to be removed
    Cleanup some templating, remove resorting of show lists and iterable container type changes
    Update some bower/npm packages,
    Fix shecdule_missed_range default to be like min_max

Fixes #3432 too [edited by @sharkykh]

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by sharkykh** [comment]
Wait up, I'll some of mine

**Comment by miigotu** [comment]
Hehe, just going to push the translations update to this next. That grunt task is fucking sweet man thanks.

File: sickbeard/config.py [hunk 1]
```diff
@@ -552,6 +552,13 @@ def change_process_automatically(process_automatically):
     return True
 
 
+def change_schedule_missed_range(coming_eps_missed_range):
+    """Makes sure the range is not more than total days since the epoch"""
+    coming_eps_missed_range = try_int(coming_eps_missed_range)
```
Comment (sharkykh) [inline sickbeard/config.py]: this is unnecessary, min_max on the next line is already doing a try_int(val, default).

Comment (miigotu) [reply sickbeard/config.py]: Yeah, dont need a helper there at all evemn

**Comment by miigotu** [comment]
done?

**Comment by sharkykh** [comment]
Just some new QC warnings that popped up. Looks like they are easy to fix, if you want

**Comment by sharkykh** [comment]
Something is still off with the minified files:
I deleted node_modules, bower_components then rebuilt everything:
```
(yarn) npm install
bower install (even though grunt does that already)
grunt
```
I have different minified files (the ones I committed)
please try to do grunt again on your end and see if something's changed.

Also, lol
> +27,590 −23,836

this is of course because of the translations

**Comment by miigotu** [comment]
Problem is that you are running yarn and it creates the files with different versions, so ofc running grunt will recreate them, because they have a different version  than yarn.

**Comment by sharkykh** [comment]
It uses the same package.json and bower.json, common sense tells me it shouldn't be different even if I'm using yarn and not npm.
I'll reinstall npm to test this theory.

**Comment by miigotu** [comment]
```bash
dusted@ROG:~/Projects/SickRage: find . -name '*shrink*'
./bower_components/bootstrap/grunt/npm-shrinkwrap.json
```

https://yarnpkg.com/en/docs/migrating-from-npm
```
If you are using an npm-shrinkwrap.json file right now, be aware that you may end up with a different set of dependencies.
Yarn does not support npm shrinkwrap files as they don’t have enough information in them to power Yarn’s more deterministic algorithm.
If you are using a shrinkwrap file it may be easier to convert everyone working on the project to use Yarn at the same time.
```

**Comment by sharkykh** [comment]
Thanks for the clarification

**Comment by sharkykh** [comment]
After removing the folders again and running on d124252701955f0a35f5370def2750158480e728,
the only changed file is `vender.min.css`

**Comment by miigotu** [comment]
If you have an urge to have everything to be up to date, the only thing that is held back atm is jquery.
https://jquery.com/upgrade-guide/3.0/

**Comment by miigotu** [comment]
```bash
dusted@ROG:~/Projects/SickRage: ncu
Using /home/dusted/Projects/SickRage/package.json
[..................] | :
All dependencies match the latest package versions :)

dusted@ROG:~/Projects/SickRage: ncu -m bower
Using /home/dusted/Projects/SickRage/bower.json

 jquery  <3.0  →  ~3.2 

Run ncu with -u to upgrade bower.json
```

**Pull request merged**